packages/ses/src/module-instance.js

```diff
@@ -278,14 +290,12 @@ export const makeModuleInstance = (
     // The module exports namespace will be wrapped in a module namespace
     // exports proxy which will serve as a "module exports namespace exotic
     // object".
-    keys(moduleProps)
+    keys(exportsProps)
       .sort()
-      .forEach(k => defProp(module, k, moduleProps[k]));
+      .forEach(k => defProp(proxiedExports, k, exportsProps[k]));
 
-    // Finally, a module exports namespace is not extensible.
-    // The module must be sealed before its namespace object can be used to
-    // resolve the promise returned by a dynamic import.
-    seal(module);
+    seal(proxiedExports);
```

Comment:
Since all the properties are accessors, it would happen to be equivalent to `freeze`, but would be better code because a reader could worry about less until they verify that all properties are accessor. Also, under maintenance, if a non-accessor is introduced this code would fail safer.